Automattic/wp-calypso
# Jetpack Pro Portal: Add prices page inside licensing section

#### Proposed Changes

This PR adds a Prices page into the Pro Portal Licensing section, so that agencies have quick access to a dynamic price list for our products.

![Screen Shot 2023-01-06 at 1 21 55 PM](https://user-images.githubusercontent.com/5528445/211074491-28879e29-443a-4e74-80c9-603330ea01dd.png)

#### Testing Instructions

Load up the Jetpack Pro Portal as an agency, and you should see a new "Prices" item in the left sidebar. Clicking that menu item should take you to a new page that looks like the screenshot. There should be no UX issues, no console errors, and the table should contract to a vertical format when viewing from a mobile size browser.

#### Pre-merge Checklist





Related to #
